qt/aqt/sound.py

```diff
@@ -177,16 +177,28 @@ def clear_queue_and_maybe_interrupt(self) -> None:
             self._stop_if_playing()
 
     def play_file(self, filename: str) -> None:
-        self.play_tags([SoundOrVideoTag(filename=os.path.basename(filename))])
+        """Play the provided path.
+
+        SECURITY: Filename may be an arbitrary path. For content coming from a collection,
+        use play_tag() instead."""
```

Comment:
To me, this implies play_tags() does some validation and enforces a media path, while it's actually exactly the same under the hood.